clients/elrond/addressMappers.go

```diff
@@ -0,0 +1,75 @@
+package elrond
+
+import (
+	"context"
+
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
+)
+
+type Erc20ToElrond struct {
```

Comment:
missing comments on exported objects

Also why do we have the structs exported?

Reply:
changed to unexported